src/libstd/ffi/c_str.rs

```diff
@@ -151,6 +151,15 @@ pub struct CStr {
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct NulError(usize, Vec<u8>);
 
+/// An error returned from `CString::into_string` to indicate that a UTF-8 error
+/// was encountered during the conversion.
+#[derive(Clone, PartialEq, Debug)]
+#[unstable(feature = "convert", reason = "recently added", issue = "0")]
```

Comment:
I've created https://github.com/rust-lang/rust/issues/29157 to connect all these features, to, so could you connect them via `issue` to that and then also update the feature name here to something like `cstring_into` or something similar?